commonmark-ext-yaml-front-matter/src/main/java/org/commonmark/ext/front/matter/YAMLFrontMatterBlock.java

```diff
@@ -0,0 +1,6 @@
+package org.commonmark.ext.front.matter;
+
+import org.commonmark.node.CustomBlock;
+
+public class YamlFrontMatterBlock extends CustomBlock {
```

Comment:
Looks like the file names aren't changed, this file is still named `YAMLFrontMatterBlock`. That's why the Travis build fails. Are you on a case-insensitive file system? Check out this [Stackoverflow question](http://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git) for how to work around that.